packages/devops/scripts/deployment/deployPackages.sh

```diff
@@ -32,6 +29,13 @@ for PACKAGE in ${PACKAGES[@]}; do
     # name (e.g. [branch-name]-api.tupaia.org -> specific-branch-api.tupaia.org)
     sed -i -e "s/\[branch-name\]/${BRANCH}/g" .env
 
+    if [[ "$BRANCH" == *-e2e ]]; then
+        # Update e2e environment variables
+        if [[ $PACKAGE == "meditrak-server" || $PACKAGE == "web-config-server" ]]; then
+            sed -i -e "s/^AGGREGATION_URL_PREFIX=dev-$/AGGREGATION_URL_PREFIX=e2e-/g" .env
+        fi
+    fi
```

Comment:
This means that all branches with an `-e2e` suffix will be connected to `e2e` aggregation by default.

Doing so in this way is probably not ideal, since the custom env logic is a bit hidden in this file. However, I felt that we don't need a more elaborate mechanism in this case.

I added a note in the e2e doc:

> By default, all instances whose subdomain ends in -e2e will be connected to the e2e aggregation servers

but I should probably add a note in a more generic doc. Deployment logic is scattered in many places (which is definitely justified), perhaps we could mitigate this with a single doc describing it.